packages/syft/src/syft/service/settings/settings_service.py

```diff
@@ -62,7 +62,7 @@ def update(
             current_settings = result.ok()
             if len(current_settings) > 0:
                 new_settings = current_settings[0].copy(
-                    update=settings.dict(exclude_unset=True)
+                    update=settings.to_dict(exclude_empty=True)
```

Comment:
Q: What is difference between  exclude_unset and exclude_empty?

Reply:
`exclude_unset` is provided by Pydantic, and it differentiates between set and unset values by a value is None or not. But this causes confusion when actually the value being set is None. Therefore we needed the notion of SyftPartialObjects and values being marked as `Empty` if they are not None.

![image](https://github.com/OpenMined/PySyft/assets/11032835/76f5bd3f-a9c0-40b8-a746-197d3c0c1e89)